### ISSUE TITLE
[ADE 508 Audit]: VA Notify Portal: 508 Audit Report for 43660 VA Notify Portal - Defect 7 - Critical

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#109089]
> [!NOTE]
> **QUICK SUMMARY:** Child ticket of https://github.com/department-of-veterans-affairs/va.gov-team/issues/109082 to track the work related to Defect 7 - Critical.

### Defect 7 | Critical | Created:
_WCAG 4.1.2 Name, Role, Value | For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies._

### Feedback from Project Team
A - Throughout the product, the name for the first link cannot be understood by assistive technology—“VA Notify logo Link Graphic” is read. This is not a link to retrieve the logo—another link that performs the same function has the name Home. (New: 03/25/2025)
[https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content__;!!K2n62VMvFw!Gn52wf00pm_uaWSi4pSvv-hyoT6od3XT5gCnSH0R2NHGb7X_irffpaDL7QnZwXjqcqoc9fhACWCMkX955vuHX9LzcHk$)[ANY PAGE] 
(A fix was added but would like confirmation)

![Image](https://github.com/user-attachments/assets/b9bc6eb1-71fb-4294-821f-51efcf76bc2a)

B - Throughout the product, the name for the Service (initially Select a service) button cannot be understood by assistive technology—the selected value is read as the name. For example, after selecting Accessibility Testing Service 1, “Accessibility Testing Service 1 (Business Contact) Button collapsed” is read. (New: 03/25/2025)

The feature that was causing this error has/will be removed and replaced with a link to a new page as part of our post-audit updates. 

(This is the same feature mentioned in Defect 2A and 2D) 

C - Throughout the product, the names for some elements cannot be understood by assistive technology. For example, on the VA Notify screen (Select a service (Accessibility Testing Service 1)), after entering test into the Template search field and pressing tab, the name for the link cannot be understood by assistive technology—no name is read. (New: 03/25/2025)

[https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content__;!!K2n62VMvFw!Gn52wf00pm_uaWSi4pSvv-hyoT6od3XT5gCnSH0R2NHGb7X_irffpaDL7QnZwXjqcqoc9fhACWCMkX955vuHX9LzcHk$)

(A fix for this issue is in place)

![Image](https://github.com/user-attachments/assets/e1832b0f-3119-45f0-93d8-3e5ef97d0311)

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._
